testdata/zk/Cheatcodes.t.sol

```diff
@@ -120,6 +129,11 @@ contract ZkCheatcodesTest is DSTest {
 
     address constant TEST_ADDRESS = 0x6Eb28604685b1F182dAB800A1Bfa4BaFdBA8a79a;
 
+    bytes32 constant STORAGE__INITIALIZABLE = bytes32(uint256(0));
```

Comment:
I'd just inline these since they are used in a single test

Reply:
It's fine to have named variables that provide context.

Reply:
Yeah, but we can just move these in the test itself. We can keep the names, and i very much agree with keeping constants named, but we don't use these in more than one test so I believe it's better to put them in the test function directly